modules/dxp/apps/analytics/analytics-reports-web/src/main/java/com/liferay/analytics/reports/web/internal/portlet/action/GetDataMVCResourceCommand.java

```diff
@@ -179,32 +181,31 @@ private JSONObject _getAuthorJSONObject(
 		InfoItemFieldValuesProvider<Object> infoItemFieldValuesProvider,
 		Locale locale, Object object) {
 
-		try {
-			InfoFieldValue<Object> infoItemFieldValue =
-				infoItemFieldValuesProvider.getInfoItemFieldValue(
-					object, "authorProfileImage");
+		InfoItemFieldValues infoItemFieldValues =
+			infoItemFieldValuesProvider.getInfoItemFieldValues(object);
 
-			JSONObject jsonObject = JSONFactoryUtil.createJSONObject(
-				String.valueOf(infoItemFieldValue.getValue(locale)));
+		InfoFieldValue<Object> authorProfileImageInfoFieldValue =
+			infoItemFieldValues.getInfoFieldValue("authorProfileImage");
 
-			long portraitId = GetterUtil.getLong(
-				_http.getParameter(
-					HtmlUtil.escape(jsonObject.getString("url")), "img_id"));
+		WebImage webImage = (WebImage)authorProfileImageInfoFieldValue.getValue(
+			locale);
 
-			if (portraitId <= 0) {
-				jsonObject.put("url", (String)null);
-			}
+		long portraitId = GetterUtil.getLong(
+			_http.getParameter(HtmlUtil.escape(webImage.getUrl()), "img_id"));
 
-			jsonObject.put(
-				"authorId", analyticsReportsInfoItem.getAuthorUserId(object));
+		String authorProfileImage = null;
 
-			return jsonObject;
+		if (portraitId > 0) {
+			authorProfileImage = webImage.getUrl();
 		}
-		catch (PortalException portalException) {
-			_log.error(portalException, portalException);
 
-			return JSONFactoryUtil.createJSONObject();
-		}
+		return JSONUtil.put(
+			"alt", webImage.getAlt()
```

Comment:
What do you think about rename this to `name`? (also in frontend)
```java
"name", webImage.getName()
```
From @carloslancha feedback.

Reply:
Awesome!!, could you open an impeding bug with this?

Reply:
Yes! Can you answer me this? https://github.com/liferay-tango/liferay-portal/pull/203#discussion_r477409702 😄